includes/card-cvv.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+?>
+	<div id="cvv-window" class="fade-in dialog-box">
+		<div id="small-x-close">x</div>
+		<div class="box d">
+			<div class="top box e">
+				<h3>Locating your Credit Card CVV/CVV2 Security Code</h3>
```

Comment:
Let's localize the strings in this file.

Reply:
Deleting this file, since we're removing this from checkout page.